### PR TITLE
Use NodeFactory by Rector

### PR DESCRIPTION
**Pull request by carusogabriel**

### CODE REVIEW

File: src/Rector/Contrib/PHPUnit/SpecificMethod/AssertTrueFalseInternalTypeToSpecificMethodRector.php [hunk 1]
```diff
@@ -110,10 +119,8 @@ private function moveFunctionArgumentsUp(MethodCall $methodCallNode): void
         unset($oldArguments[0]);
 
         $methodCallNode->args = array_merge([
-            new Arg(new String_(
-                $this->oldMethodsToTypes[$isFunctionName]
-            )),
-            new Arg($argument),
+            $this->nodeFactory->createArg(new String_($this->oldMethodsToTypes[$isFunctionName])),
```
Comment (carusogabriel) [inline src/Rector/Contrib/PHPUnit/SpecificMethod/AssertTrueFalseInternalTypeToSpecificMethodRector.php]: @TomasVotruba As we have `createTrueConstant` and like methods in `NodeFactory`, could we have `createString`? :thinking:

File: src/Rector/Contrib/PhpParser/SetLineRector.php [hunk 2]
```diff
@@ -29,10 +28,19 @@
      */
     private $identifierRenamer;
 
-    public function __construct(MethodCallAnalyzer $methodCallAnalyzer, IdentifierRenamer $IdentifierRenamer)
```
Comment (carusogabriel) [inline src/Rector/Contrib/PhpParser/SetLineRector.php]: @TomasVotruba Is there a Fixer for `$identifierRenamer` instead of `$IdentifierRenamer`?

Comment (TomasVotruba) [reply src/Rector/Contrib/PhpParser/SetLineRector.php]: You can extend this one: https://github.com/Symplify/Symplify/blob/master/packages/CodingStandard/README.md#property-name-should-match-its-type-if-possible-wrench

**Comment by TomasVotruba** [comment]
Thank you

**Pull request merged**